x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/inline_radio_group.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useEuiTheme, EuiRadioGroup, type EuiRadioGroupProps } from '@elastic/eui';
+import { css } from '@emotion/react';
+
+type RadioGroupProps = Pick<EuiRadioGroupProps, 'onChange' | 'options' | 'idSelected'>;
+
+type Props = RadioGroupProps & {
+  size?: 's' | 'm';
+};
+
+export const InlineRadioGroup = ({ idSelected, size, options, onChange }: Props) => {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <EuiRadioGroup
```

Comment:
not sure which one is better, extending `EuiRadioGroup` and tweaking its internal styles which feels hacky and possibly fragile

or [composing `EuiButton` and `EuiRadio`](https://github.com/elastic/kibana/blob/fea9d98d7e0c67f6ffb3ef5b018203d753c08c3e/x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/inline_radio_group.tsx) in a non HTML semantic, which itches wrong and feels like a shortcut

i went with the first one because i ultimately i think it's unlikely this component breaking will go un-noticed in development after some eui changes. 

thoughts and opinions are welcome